modules/reindex/build.gradle

```diff
@@ -23,6 +25,19 @@ esplugin {
   hasClientJar = true
 }
 
+Task integTestNetty4 = tasks.create("integTestNetty4", RestIntegTestTask) {
```

Comment:
I [tried](https://github.com/elastic/elasticsearch/pull/19526#discussion_r71822933) that. For now I will just leave the way that I have.